src/openms/include/OpenMS/CHEMISTRY/EmpiricalFormula.h

```diff
@@ -248,6 +253,10 @@ namespace OpenMS
     inline ConstIterator begin() const { return formula_.begin(); }
 
     inline ConstIterator end() const { return formula_.end(); }
+    
+    inline Iterator begin() { return formula_.begin(); }
```

Comment:
These `inline`s are superfluous, the methods defined inside class are implicitly inline already. (And these are just hints anyway, it's up to compiler to decide.)